AmazonChimeSDK/AmazonChimeSDK/internal/ingestion/database/DirtyEventSQLiteDao.swift

```diff
@@ -29,6 +29,7 @@ class DirtyEventSQLiteDao: DirtyEventDao {
                let dataStr = event["data"] as? String,
                let ttl = event["ttl"] as? Int64,
                let data = dataStr.data(using: .utf8),
+               // TODO: This will skip the item from processing by EventBuffer if decoding fails, should always return the event so that EventBuffer can delete it from database.
```

Comment:
Could you explain this a bit?

Reply:
If decode fails at line 33, this event will not be put int `events`, or returned to event buffer for consuming, so that the event buffer will not get a chance to check the `ttl` and delete it from database.